src/libmongoc/doc/ref/packages.rst

```diff
@@ -0,0 +1,200 @@
+##############################
+Package Installation Reference
+##############################
+
+.. _EPEL: https://docs.fedoraproject.org/en-US/epel/
+.. _conan: https://conan.io/
+.. _vcpkg: https://vcpkg.io/
+
+|libbson| and |libmongoc| are available from several package management tools
+on a variety of systems.
+
+.. important::
+
+  The third-party packages detailed here are not directly controlled via the
+  |mongo-c-driver| maintainers, and the information found here may be incomplete
+  or out-of-date.
+
+
+Package Names and Availability
+******************************
+
+This table details the names and usage notes of such packages.
+
+.. seealso::
+
+  For a step-by-step tutorial on installing packages, refer to
+  :doc:`/learn/get/installing`.
+
+
+.. list-table::
+  :name: Available Packages
+  :header-rows: 1
+  :align: left
+  :widths: auto
+
+  - - Packaging Tool
+    - Platform(s)
+    - |libbson| package(s)
+    - |libmongoc| package(s)
+    - Notes
+
+  - - :index:`APT <!pair: APT; package names>` (``apt``/``apt-get``)
+    - Debian-based Linux distributions (:index:`Debian`, :index:`Ubuntu`, Linux
+      Mint, etc.)
+    - ``libbson-1.0-0``, ``libbson-dev``,
+      ``libbson-doc``
+    - ``libmongoc-1.0-0``, ``libmongoc-dev``,
+      ``libmongoc-doc``
+    - The ``libmongoclient`` packages are for the C++ database driver and will
+      transitively depend on ``libmongoc``.
```

Comment:
```suggestion
    - .. empty cell
```

libmongoclient appears to refer to the legacy C++ driver and is not maintained by us.

[mongo-cxx-driver](https://packages.debian.org/source/experimental/mongo-cxx-driver) is in experimental. Suggest removing mention of the C++ driver.